src/main.cpp

```diff
@@ -23,13 +21,14 @@ void setup(void) {
 
   Network::init();
   if (Network::getMode() == Network::MODES::ST) {
-    if (Settings::use_mqtt && Mosquitto::init(messageHandler)) {
+    if (Settings::use_mqtt && Mosquitto::init(strip)) {
```

Comment:
I think we should remove this `use_mqtt` flag.. for what I'm seeing aurora can't do much without it.

Reply:
There's many use cases where we don't use mqtt at all, for instance the web UI doesn't require it, this allows for optional usage of mqtt